streamer/src/quic.rs

```diff
@@ -136,7 +136,7 @@ impl NotifyKeyUpdate for EndpointKeyUpdater {
 }
 
 #[derive(Default)]
-pub struct StreamStats {
+pub struct ConnectionStats {
```

Comment:
This is not all about Connection. ConnectionStats gives an impression it is for a connection. Maybe originally we wanted to name  it to StreamerStats.

Reply:
Will rename to `StreamerStats`

Reply:
all done

Reply:
If we want to go this route, why not rename to QuicStreamerStats instead, since these are specific to the quic server and not the UDP server cc @lijunwangs

Reply:
Well,  if the plan to remove UDP any time soon, I think it is fine to have the name without Quic. Otherwise, if there is UDP `Streamer` somewhere (not sure), adding Quic will make it more specific and will be good.